image/types/src/lib.rs

```diff
@@ -26,6 +26,7 @@ use zerocopy::{AsBytes, FromBytes};
 pub const MANIFEST_MARKER: u32 = 0x4E414D43;
 pub const VENDOR_ECC_KEY_COUNT: u32 = 4;
 pub const VENDOR_LMS_KEY_COUNT: u32 = 4;
+pub const OWNER_LMS_KEY_COUNT: u32 = 4;
```

Comment:
Why do we need 4 Owner LMS keys? Shouldn't it just be 1.

Reply:
Because LMS keys are stateless it is dangerous to just have one single shard. This is for disaster-recovery purposes.
We dont need to implement revocation of  owner LMS keys if we dont want to (and don't have fuses for it).

Reply:
Owner key must be 1.

Reply:
Reverting to 1 now. will revisit later.